### PR TITLE
55066 html refactor

### PR DESCRIPTION
**Pull request by tshedor**
This is a mammoth PR that organizes the SCSS and cleans up some of the template files while maintaining support for apps pre 1.3. I haven't encountered any bugs relating to these changes on the front end.

### CODE REVIEW

File: app/views/fae/options/_form.html.slim [hunk 1]
```diff
@@ -8,12 +8,10 @@
...
-  .main_content-section
-    section.main_content-section
-      .main_content-section-area
-        = fae_input f, :title, label: "Project Name"
-        = fae_input f, :time_zone, wrapper_class: 'select'
-        = fae_input f, :live_url, label: 'Live URL <h6>Include http:// for external links.</h6>'.html_safe
-        = fae_input f, :stage_url, label: 'Stage URL <h6>Include http:// for external links.</h6>'.html_safe
-        = fae_image_form f, :logo, required: true
-        = fae_image_form f, :favicon
+  section.main_content-section-area
```
Comment (jamesmk) [inline app/views/fae/options/_form.html.slim]: :heart:

**Comment by jamesmk** [comment]
@tshedor: did you convert all dummy app views to the new structure? I think we should keep one section with the old markup to test legacy support.

**Comment by jamesmk** [comment]
What do you think about duplicating the release section 3 times: one in current slim, one in current erb and one in old slim? That way we'd have documented examples and a robust index/form to QA.

**Comment by tshedor** [comment]
Good call on the legacy support. I'll do that thing with the release section. What do you mean by "old slim"?

**Comment by jamesmk** [comment]
The old structure you just refactored, but in slim. It's currently in erb, which I think we should have as an example only up-to-date on the structure. Make sense?

**Comment by tshedor** [comment]
@jamesmk `releases/index.html.erb` and `wines/index.html.slim` use the pre-refactor HTML and classes. All other views are up-to-date (new slim and new old ERB).

**Comment by tshedor** [comment]
@jamesmk [Added in 4fc376b](https://github.com/wearefine/fae/commit/4fc376b01b7a87ccd9ebc278163a924ec1ba074e), you can change the views of `release#index`, `release#new`, and `release#edit` by appending `?legacy=<old_erb|old_slim>` to the URL

**Comment by jamesmk** [comment]
closing in favor of #37

**Pull request closed without merging**